views/default/js/live_chat.js

```diff
@@ -5,7 +5,7 @@ define(function(require) {
 	var messageTemplate = require('text!live_chat/message.html');
 	var moduleTemplate = require('text!live_chat/module.html');
 
-	var users = {};
+	var chat_guids = {};
```

Comment:
Can you document here that this is an associative array of `user_guid => chat_guid`? It took me a moment to realize what's happening.